src/cache/cache_las.c

```diff
@@ -787,6 +788,11 @@ __wt_las_insert_block(WT_CURSOR *cursor,
 		__las_insert_block_verbose(session, btree, multi);
 	}
 
+	las_btree = S2BT_SAFE(session);
```

Comment:
I've had to use `S2BT_SAFE` here because the `dhandle` member of session is `NULL` in a few of the Python suite tests. I wasn't sure whether this was ok/expected so I figured I'd just open this up and gauge reactions before investigating further. I've tried moving this block before the `err` tag to check whether it's the cases where we jump with `WT_ERR` and fail to write to the LAS that is causing this but that doesn't seem to be the what's happening.

Reply:
8150be1